### ISSUE TITLE
Stackable paper-toast

### ISSUE THREAD

**Issue by raffaeu** [Polymer/paper-toast#25]
At the moment if we want to display multiple toasts we simply have to swap the content of the current one. Example, I have a page with the following `paper-toast` in it:

```
<!-- Mini Message center -->
<paper-toast id="toast" class="fit-bottom" duration="0">
    <paper-button id="toastButton" onclick="toast.toggle()">Close</paper-button>
</paper-toast>
```

And this is the code to display a Message:

```
var toast = document.querySelector('#toast');
toast.show({ text: data.detail.message });
```

Now, if I call again this method while the Toast is already open, it simply displays a new message. My intent is to have multiple stackable toasts, each one dismissable, like this example:

![4-29-2016 4-26-33 pm](https://cloud.githubusercontent.com/assets/1304736/14917407/975f3b1a-0e27-11e6-89eb-ed4d1a2cad93.png)

I thought to use a sort of `<template is="dom-repeat">` and repeat an entire`<paper-toast>` element for each one of my messages and stack them one on top of each other. Is this the correct approach?

**Comment by red15** [comment]
@raffaeu note that the [Google material design](https://material.google.com/components/snackbars-toasts.html#snackbars-toasts-usage) philosophy says to not show stacks of them but show them sequentially:

> They should not be persistent or be stacked, as they are above other elements on screen.

**Comment by chrisgiffy** [comment]
I need to do something exactly like this, however, my messages are unpredictable and if I receive another message before the first one disappears I need to stack them up.

**Comment by red15** [comment]
Consider bumping a new notification (which will replace the current notification) with a "multiple unread notifications" message, then link that notification to an "inbox" view which allows you to simply use a list to show the unread notifications (think like github's own [notification view](https://github.com/notifications) )

This should save you from having to code against the paper-toast design and would still give you proper indication to the user of what's going on.

**Comment by raffaeu** [comment]
Agree, actually it should be like a message broadcasting service where
messages are collected in a queue, including for example errors and
warnings and then could be easily browsed in a different sidebar, like
windows 10 notification center does.
